docs/design/chart_format.md

```diff
@@ -66,19 +59,18 @@ We assume that we are developing a technology that will enjoy widespread use amo
 * Advanced users may be interested in, and willing to, learn advanced features to build much more interesting charts
 
 
-Based on these, our design goal is to make it **easy to find, use, author, release, and maintain charts**, while still making it possible for advanced users to build sophisticated charts. Furthermore, we have begun the process of **attaching provenance to charts**.
+Based on these, our design goal is to make it **easy to find, use, author, release, and maintain charts**, while still making it possible for advanced users to build sophisticated charts.
 
+This design does not lose any of the functionality of Deployment Manager (DM), and which loses only Helm functionality that we believe is disposable. But both tools have changed substantial portions of their chart-handling logic.
```

Comment:
This design is based on the integration of Deployment Manager (DM) and Helm, formerly at github.com/deis/helm. It does not lose any of the functionality of DM, and loses only Helm functionality that we believe is disposable. Substantial portions of the chart and template handling logic from the original implementations of these two tools have changed as a result of this integration. The new chart format, described in this document, is one of the primary drivers of those changes.